src/Symfony/Component/Security/Core/Tests/Encoder/BCryptPasswordEncoderTest.php

```diff
@@ -42,7 +42,7 @@ public function testCostAboveRange()
     public function testCostInRange()
     {
         for ($cost = 4; $cost <= 31; ++$cost) {
```

Comment:
this should use a data provider, to ensure that 31 is tested even if we have a failure for 4